agent/lib/kontena/network_adapters/ipam_client.rb

```diff
@@ -73,7 +73,12 @@ def release_address(network, address)
       debug "response: #{response.status}/#{response.body}"
       JSON.parse(response.body)
     rescue Excon::Errors::HTTPStatusError => error
-      handle_error_response(error)
+      if error.response.status == 409
+        # IPAM return 409 in case the address still responds to ping, case zombies
+        warn error.response.body
```

Comment:
Would it make more sense to handle this a bit higher up in the stack, in the `Weave` actor?

```ruby
def remove_container(...)

rescue IpamError => error
  # leave it for later cleanup
  warn "Failed to release container=#{container_id} from network=#{overlay_network} at cidr=#{overlay_cidr}: #{error}"
end
```